test/integration/backward_compatible/parallel/serialization/compact/CompactTest.js

```diff
@@ -78,7 +78,8 @@ describe('CompactTest', function () {
     before(async function () {
```

Comment:
Should we update other failing tests like SQL tests about compact etc.?

Reply:
Can you check all failing tests one by one? You need to be careful with tests with the same suite name / file name.